package-lock.json

```diff
@@ -1,943 +1,8 @@
 {
   "name": "kurl",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Can you explain why this is changing? Doesn't NPM set this?

Reply:
Yes, the changes in this PR happened by running npm install only.